app/sequencescape_excel/sequencescape_excel/column.rb

```diff
@@ -79,6 +79,10 @@ def unlocked?
       unlocked
     end
 
+    def style
+      [unlocked? ? :unlocked : :locked, type]
```

Comment:
It doesn't have any specified styles. At one point I was returning nil, but then just ended up needing to add support for missing styles anyway, so it felt more useful to pass through a token that we could potentially hang behaviour off.

In particular, there is an issue with pasted content being automatically locked, which I think is related to properties of the 'default' style. So we might be better explicitly locking cells, rather than unlocking them.